### DISCUSSION TITLE
Copilot only responding with short responses, if at all.

### DISCUSSION THREAD

**Discussion by Nebula-Developer** [community/community#46816] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When I have recently been using Copilot I have noticed the responses it will return have exponentially decreased in length, and it will commonly load for around 500ms, then stop, not creating a suggestion at all.

This did not used to occur, and has only started happening in the past month or so.
I thought it was due to my internet, and it timing out (I have somewhat slow internet), but it happens on the three networks I have tested.

Viewing the output of copilot in VSCode, it will echo this:
```
[INFO] [default] [2023-02-09T05:13:06.478Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2023-02-09T05:13:06.812Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 333 ms
[INFO] [streamChoices] [2023-02-09T05:13:06.816Z] solution 0 returned. finish reason: [content_filter]
[INFO] [streamChoices] [2023-02-09T05:13:06.817Z] request done: headerRequestId: [65db439c-7eb5-4b28-88a5-50eb474f9120] model deployment ID: [x625ba4f78aa8]
```

I am unsure whether content_filter is the culprit, but nevertheless thought I would post this on here to see if anyone has any ideas. Keep in mind this is happening over a variety of devices I have used.

**Comment by Akash1134** [comment]
Hi @Nebula-Developer , Sorry that you're facing this issue. Our team is aware of it and working towards a fix. I'd recommend that you look at this [discussion](https://github.com/orgs/community/discussions/40522#discussioncomment-4701470) for more details.

**Reply by Nebula-Developer** [reply]
Cheers. Hope everything gets sorted out soon.

**Comment by davecheney** [comment]
@Nebula-Developer are you able to share a sample of the file you were working on that triggered the `content_filter` response.

**Comment by DurandSacha** [comment]
Dans l'attente d'un fix, il faudrait améliorer la quantité de code (prompt), afin de donner tous les éléments minimum pour que copilot puisse faire son travail.
